CMakeLists.txt

```diff
@@ -66,7 +66,9 @@ global_target_config()
 
 include(CMake/install_config.cmake)
 
-add_subdirectory(wrappers)
+if (BUILD_WRAPPERS)
```

Comment:
@oiramario , thanks for the proposal,
However the requested flag will create compatibility issues with the existing flags and lead to non-deterministic behavior, e.g.
>cmake ..  -DBUILD_PYTHON_BINDINGS=**false**   -DBUILD_CV_EXAMPLES=**false**  -DBUILD_WRAPPERS=**true**

What would be the expected result in this case ?  it is either ambiguous to undefined.

Reply:
we could:
if BUILD_PYTHON_BINDINGS==tru || DBUILD_CV_EXAMPLES==true then
BUILD_WRAPPERS=true
endif

The CMakeLists.txt in wrappers is good enough.
I don't think it's necessary now , so ignore this.